lib/ansible/modules/cloud/ovirt/ovirt_template.py

```diff
@@ -426,6 +438,39 @@ def _get_vnic_profile_mappings(module):
     return vnicProfileMappings
 
 
+def _template_subversion(module, templates_service, templates_module, connection):
+    changed = False
+    resp = None
+    version = module.params.get('version')
+    if version.get('number') is not None:
+        templates = templates_service.list()
```

Comment:
Can you please add this logic to some method called `find_template` so you don't have to reimplenet the logic of update/create.